cartographer_ros/cartographer_ros/color_manager.cc

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2017 The Cartographer Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cartographer_ros/color_manager.h"
+
+#include <cmath>
+
+#include "glog/logging.h"
+
+namespace cartographer_ros {
+
+constexpr double kColorManagerInitialHue = 0.69;
```

Comment:
`ColorManager` should probably be removed from the name. The file should probably also just be called `color.cc`.